homeassistant/components/fritzbox_netmonitor/sensor.py

```diff
@@ -1,86 +1,100 @@
 """Support for monitoring an AVM Fritz!Box router."""
-from datetime import timedelta
-import logging
 
-from fritzconnection.core.exceptions import FritzConnectionException
-from fritzconnection.lib.fritzstatus import FritzStatus
 from requests.exceptions import RequestException
 import voluptuous as vol
 
 from homeassistant.components.sensor import PLATFORM_SCHEMA
-from homeassistant.const import CONF_HOST, CONF_NAME, STATE_UNAVAILABLE
+from homeassistant.config_entries import SOURCE_IMPORT
+from homeassistant.const import CONF_HOST, STATE_UNAVAILABLE
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.entity import Entity
 from homeassistant.util import Throttle
 
-_LOGGER = logging.getLogger(__name__)
-
-CONF_DEFAULT_NAME = "fritz_netmonitor"
-CONF_DEFAULT_IP = "169.254.1.1"  # This IP is valid for all FRITZ!Box routers.
-
-ATTR_BYTES_RECEIVED = "bytes_received"
-ATTR_BYTES_SENT = "bytes_sent"
-ATTR_TRANSMISSION_RATE_UP = "transmission_rate_up"
-ATTR_TRANSMISSION_RATE_DOWN = "transmission_rate_down"
-ATTR_EXTERNAL_IP = "external_ip"
-ATTR_IS_CONNECTED = "is_connected"
-ATTR_IS_LINKED = "is_linked"
-ATTR_MAX_BYTE_RATE_DOWN = "max_byte_rate_down"
-ATTR_MAX_BYTE_RATE_UP = "max_byte_rate_up"
-ATTR_UPTIME = "uptime"
-
-MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=5)
-
-STATE_ONLINE = "online"
-STATE_OFFLINE = "offline"
-
-ICON = "mdi:web"
+from .const import (
+    ATTR_BYTES_RECEIVED,
+    ATTR_BYTES_SENT,
+    ATTR_EXTERNAL_IP,
+    ATTR_IS_CONNECTED,
+    ATTR_IS_LINKED,
+    ATTR_MAX_BYTE_RATE_DOWN,
+    ATTR_MAX_BYTE_RATE_UP,
+    ATTR_TRANSMISSION_RATE_DOWN,
+    ATTR_TRANSMISSION_RATE_UP,
+    ATTR_UPTIME,
+    DEFAULT_HOST,
+    DOMAIN,
+    ICON,
+    LOGGER,
+    MANUFACTURER,
+    MIN_TIME_BETWEEN_UPDATES,
+    STATE_OFFLINE,
+    STATE_ONLINE,
+)
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
-        vol.Optional(CONF_NAME, default=CONF_DEFAULT_NAME): cv.string,
-        vol.Optional(CONF_HOST, default=CONF_DEFAULT_IP): cv.string,
+        vol.Optional(CONF_HOST, default=DEFAULT_HOST): cv.string,
     }
 )
```

Comment:
Can I use `cv.deprecated()` in this PLATFORM_SCHEMA to inform users that this will be removed in a certain version?